libraries/testkit/user.py

```diff
@@ -27,6 +27,8 @@ def __init__(self, target, db, name, password, channels):
         self.target = target
 
         self._session = requests.Session()
+        adapter = requests.adapters.HTTPAdapter(pool_connections=100, pool_maxsize=100)
```

Comment:
This is the fix for all of those Connection Pool errors you see from time to time. This is sufficient for the scale of those tests. Also, a good read is https://laike9m.com/blog/requests-secret-pool_connections-and-pool_maxsize,89/. I recommend checking it out.